### PR TITLE
Book model, book repository, book service

### PR DESCRIPTION
**Pull request by pastcwdt**

### CODE REVIEW

**Review by mmosii — APPROVED** [review]
LGTM. Great job!

File: src/main/java/past/cwdt/bookstore/model/Book.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Table(name = "books")
+@Data
```
Comment (liliia-ponomarenko) [inline src/main/java/past/cwdt/bookstore/model/Book.java:14]: Lombok is a great library but be [attentive when using it with Entity](https://jpa-buddy.com/blog/lombok-and-jpa-what-may-go-wrong/) JFYI

File: src/main/java/past/cwdt/bookstore/repository/impl/BookRepositoryImpl.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import org.springframework.stereotype.Repository;
+import past.cwdt.bookstore.exceptions.DataProcessingException;
+import past.cwdt.bookstore.model.Book;
+import past.cwdt.bookstore.repository.BookRepository;
+
+@Repository
+public class BookRepositoryImpl implements BookRepository {
+    private final SessionFactory sessionFactory;
+
+    @Autowired
```
Comment (liliia-ponomarenko) [inline src/main/java/past/cwdt/bookstore/repository/impl/BookRepositoryImpl.java:17]: As of Spring Framework 4.3, an @Autowired annotation on such a constructor is no longer necessary if the target bean defines only one constructor to begin with. See [note here ](https://docs.spring.io/spring-framework/reference/core/beans/annotation-config/autowired.html)Let’s check it ;)

**Review by liliia-ponomarenko — APPROVED** [review]
Great! Check comments before merging ;)

**Pull request merged**